composer.json

```diff
@@ -16,6 +16,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8"
+        "phpunit/phpunit": "^4.8.10|^5.0"
```

Comment:
Should probably be `^4.8.10 || ^5.0`?

Also, PHPUnit 5 emits several warnings on PHP 7, can you address these here? See also #93 and #107.